ietf-wdm-interface.tree

```diff
@@ -40,45 +40,51 @@ module: ietf-ext-xponder-wdm-if
              +--ro tx-polarization-power-difference?   power-ratio
              +--ro polarization-skew?                  decimal-2
   augment /if:interfaces/if:interface:
-    +--rw optIfOChRsSs
-       +--rw supported-modes!
+    +--rw wdm-interface
+       +--ro supported-modes!
        |  +--ro supported-mode* [mode-id]
-       |     +--ro mode-id                         string
+       |     +--ro mode-id                           string
        |     +--ro (mode)
        |        +--:(G.698.2)
-       |        |  +--ro standard-mode?            standard-mode
-       |        |  +--ro line-coding-bitrate*      identityref
-       |        |  +--ro min-central-frequency?    frequency-thz
-       |        |  +--ro max-central-frequency?    frequency-thz
-       |        |  +--ro transceiver-tunability?   frequency-ghz
-       |        |  +--ro tx-channel-power-min?     power-dbm
-       |        |  +--ro tx-channel-power-max?     power-dbm
-       |        |  +--ro rx-channel-power-min?     power-dbm
-       |        |  +--ro rx-channel-power-max?     power-dbm
-       |        |  +--ro rx-total-power-max?       power-dbm
+       |        |  +--ro standard-mode?              standard-mode
+       |        |  +--ro line-coding-bitrate*        identityref
+       |        |  +--ro transmitter-tuning-range
+       |        |  |  +--ro min-central-frequency?    frequency-thz
+       |        |  |  +--ro max-central-frequency?    frequency-thz
+       |        |  |  +--ro transceiver-tunability?   frequency-ghz
```

Comment:
What is the meaning of transceiver-tunability? I would have thought it would be a boolean. Alternatively it could be a tunability-granularity information, but that isn't clear from the name.